openupgrade_scripts/scripts/sale/16.0.1.2/pre-migration.py

```diff
@@ -0,0 +1,56 @@
+from openupgradelib import openupgrade
+
+_models_renames = [
+    (
+        "sale.payment.acquirer.onboarding.wizard",
+        "sale.payment.provider.onboarding.wizard",
+    ),
+]
+
+_renames_xmlids = [
+    (
+        "sale.sale_payment_provider_onboarding_wizard_rule",
+        "sale.sale_payment_acquirer_onboarding_wizard_rule",
+    ),
```

Comment:
this does not work, as the old and new name are inverted. but anyway, it would be wrong (as explained in my comment on `upgrade_analysis_work.txt`: the only thing to do is to delete the old rule (`sale.sale_payment_acquirer_onboarding_wizard_rule`).